packages/xo-web/src/xo-app/self/index.js

```diff
@@ -494,6 +498,12 @@ export class Edit extends Component {
                 </Col>
               </Row>
             </div>
+            <div className='mt-1'>
+              <input checked={state.share} type='checkbox' onChange={this.toggleState('share')} />{' '}
+              <label>
+                <strong>{_('shareVmsByDefault')}</strong>
+              </label>
+            </div>
```

Comment:
Wouldn't it be nice to have the label clickable?
```suggestion
            <div className='mt-1'>
              <label>
                <input checked={state.share} type='checkbox' onChange={this.toggleState('share')} />
                &nbsp;
                <strong>{_('shareVmsByDefault')}</strong>
              </label>
            </div>
```